docs/cloud/how-to-create-a-ticket-for-temporal-support.md

```diff
@@ -0,0 +1,31 @@
+---
+id: how-to-create-a-ticket-for-temporal-support
+title: How to create a ticket for Temporal Support
+sidebar_label: Support ticket
+description: To request assistance from Temporal Support, create a ticket in Zendesk.
+tags:
+  - explanation
+---
+
+:::info
+
+This procedure applies only to Temporal Cloud customers whose contracts include paid support.
+If you need assistance and don't have paid support, post your request in the [Temporal Community Forum](https://community.temporal.io) or the [Temporal Slack workspace](https://t.mp/slack).
+
+:::
+
+To request assistance from Temporal Support, create a ticket in Zendesk:
+
+1. Go to [support.temporal.io](https://support.temporal.io/).
+1. Use your Temporal Support credentials to sign in.
+1. Choose **Create a ticket**.
+1. On the **Submit a request** page, choose your issue.
+   Unless your request involves one of the specific areas listed, choose **Submit a Ticket**.
+1. In the form, enter the details of your request.
+   **Subject** and **Description** are required.
+1. If you specify **Priority** (available only on the default form), follow these guidelines:
+   - Select **Normal** for most issues.
+   - Select **High** only for issues to which your service-level agreement (SLA) applies.
+     If you're not sure, check with your Temporal administrator.
```

Comment:
whats a temporal administrator? an internal person to the customer or someone at temporal?

Reply:
Presumably someone at the customer company who has knowledge of the applicable SLA. What would be a clearer way to express this? Or should I remove the sentence?

Reply:
Let's bypass the issue and replace the sentence with "If you're not sure, select **Normal**." Either way, Support gets the ticket, and you can elevate it if needed.